### PR TITLE
Document --disable-error-code cli and config option

### PR DESCRIPTION
**Pull request by amikrop**
Added the --disable-error-code cli option and disable_error_code config option to the documentation.

### CODE REVIEW

File: docs/source/command_line.rst [hunk 1]
```diff
@@ -480,6 +480,11 @@ of the above sections.
            # 'items' now has type List[List[str]]
            ...
 
+.. option:: --disable-error-code CODE
```
Comment (hauntsaninja) [inline docs/source/command_line.rst:483]: The command line option is already documented on line 556.

Comment (amikrop) [reply docs/source/command_line.rst]: OK removed the duplicate entry I added

File: docs/source/config_file.rst [hunk 2]
```diff
@@ -529,6 +529,12 @@ Miscellaneous strictness flags
     Allows variables to be redefined with an arbitrary type, as long as the redefinition
     is in the same block and nesting level as the original definition.
 
+.. confval:: disable_error_code
+
+    :type: comma-separated list of strings
+
+    Causes mypy to suppress errors with given error codes.
```
Comment (hauntsaninja) [inline docs/source/config_file.rst:536]: One nit: I'd steal the language from the command line docs here (for consistency, and for the mention of "globally").
```suggestion
    Allows disabling one or multiple error codes globally.
```

(More generally, duplication between command line docs and config docs seems like a bit of a maintenance headache)

**Review by hauntsaninja — APPROVED** [review]
Thanks again!

**Comment by amikrop** [comment]
> Thanks again!

You are welcome.

**Pull request merged**